app.R

```diff
@@ -262,7 +261,19 @@ shinyApp(
         # Use scatterplot or lines depending on the x var.
         {
           if (input$x_var %in% c(input$facet_vars, "geo_value", "forecaster", "ahead")) {
-            . + geom_point(aes(size = n / 100)) + expand_limits(size = 0)
+            scale_factor_fcast <- length(input$selected_forecasters) * 2
+            scale_factor_facet <- length(input$facet_vars) * 5
+            scale_factor_geo <- ("geo_value" %in% input$facet_vars) * (60 - length(input$excluded_geo_values))
+            scale_factor_geo_x <- ("geo_value" %in% input$x_var) * 10
+            scale_factor_facet_fcast <- ifelse("forecaster" %in% input$facet_vars, length(input$selected_forecasters), 0) * 5
```

Comment:
note: fudge factors for scaling in different situations. A higher number makes all points smaller on the plot.